src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -93,7 +93,7 @@ internal abstract partial class DefinitionItem
         /// Additional locations to present in the UI.  A definition may have multiple locations 
         /// for cases like partial types/members.
         /// </summary>
-        public ImmutableArray<DocumentSpan> SourceSpans { get; }
+        public ImmutableArray<SerializableDocumentSpan> SourceSpans { get; }
```

Comment:
this is the core part of the change that ensures that anything holding onto a DefItem doesn't root the solution.